mqtt/client.go

```diff
@@ -26,7 +26,7 @@ var DefaultClient = Client{
 	server:    "tcp://mqtt.meshtastic.org:1883",
 	username:  "meshdev",
 	password:  "large4cats",
-	topicRoot: "msh", //TODO: this will need to change
+	topicRoot: "msh/2", //TODO: this will need to change
```

Comment:
@crypto-smoke - does this change seem right to you ? Perhaps the consumer should specify the version in a separate field ? Or we could hardcode it.

Reply:
That's just some cruft left in from before. Hardcoding all the things. Now there are acutally multiple root topics, separated by region. the `2/c` part of the topic should probably be implied, wihch is not how the phone apps do it. IOS app shows just `msh` for the root topic but was using `msh/2/c`

Reply:
I'm actually not too sure we can do a `MQTTProtoTopic` with the current implementation - not without modifying `(c *Client) Handle` as this calls into `GetFullTopicForChannel`

Reply:
Im okay with modifying that, it was just a quick hack to make my stuff work.

Reply:
Addressed in a4ddbb9